samples/client/petstore/php/OpenAPIClient-php/lib/Api/AnotherFakeApi.php

```diff
@@ -64,19 +64,47 @@ class AnotherFakeApi
      */
     protected $headerSelector;
 
+    /**
+     * @var Host index
+     */
+    protected $hostIndex;
+
     /**
      * @param ClientInterface $client
      * @param Configuration   $config
      * @param HeaderSelector  $selector
+     * @param int             $host_index
      */
     public function __construct(
         ClientInterface $client = null,
         Configuration $config = null,
-        HeaderSelector $selector = null
+        HeaderSelector $selector = null,
+        $host_index = 0
```

Comment:
How about the `$host_index` generates only if `servers` exists in pathItems?

I just wondering that the `$host_index` is defined but unused. 👀

Reply:
Yes, very good suggestion. My intention is to make the constructor consistent across different API classes.

What about updating the doc string to explain it cleary that it's optional and only needed when the endpoint has multiple servers/hosts defined?

Reply:
I agree with that. 👍

Reply:
Pushed some enhancements based on your feedback.